tools/serve/serve.py

```diff
@@ -447,8 +448,17 @@ def make_hosts_file(config, host):
     for domain in config.domains_set:
         rv.append("%s\t%s\n" % (host, domain))
 
-    for not_domain in config.not_domains_set:
-        rv.append("0.0.0.0\t%s\n" % not_domain)
+    # Windows interpets the IP address 0.0.0.0 as "non-existent, making it an
```

Comment:
Needs a closing double quote. (Also, where's the quote from? I can't find this documented anywhere?)

Reply:
I've added the double quote in a fixup! commit. I'll respond to your question in the main thread so the conversation doesn't get lost when GitHub collapses this comment.